recipes-support/px4-fw/px4-fw_0.1.bb

```diff
@@ -15,4 +15,7 @@ do_install() {
     install -d ${D}${sysconfdir}/aerofc/px4
     install -m 0755 ${S}/nuttx-aerofc-v1-default.px4 ${D}${sysconfdir}/aerofc/px4
     install -m 0644 ${S}/LICENSE.md ${D}${sysconfdir}/aerofc/px4
+    # Add symbol link to the old directory
+    install -d ${D}${sysconfdir}/px4-fw
+    ln -s ${sysconfdir}/aerofc/px4/nuttx-aerofc-v1-default.px4 ${D}${sysconfdir}/px4-fw/nuttx-aerofc-v1-default.px4
```

Comment:
ln -s ${sysconfdir}/aerofc/px4/  ${D}${sysconfdir}/px4-fw
then you can remove line 19.